package.json

```diff
@@ -20,28 +20,29 @@
   "typings": "index.d.ts",
   "homepage": "https://jhipster.github.io",
   "peerDependencies": {
-    "@angular/core": "^4.1.3",
-    "@angular/http": "^4.1.3"
+    "@angular/core": "5.0.0",
```

Comment:
use `^` versions as these are peerDependencies else we would have to keep them in sync. These dependencies will be decided by the generated app